src/System.Data.SqlClient/src/System/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -98,6 +98,142 @@ internal static string ConvertToString(object value)
                 throw ADP.ConvertFailed(value.GetType(), typeof(String), e);
             }
         }
+        #region <<PoolBlockingPeriod Utility>>
+        const string PoolBlockingPeriodAutoString = "Auto";
+        const string PoolBlockingPeriodAlwaysBlockString = "AlwaysBlock";
+        const string PoolBlockingPeriodNeverBlockString = "NeverBlock";
```

Comment:
Rather than naming these consts like this, could you just use `nameof(PoolBlockingPeriod.Auto)`, etc., at the call sites where they're used?